src/kOS/Control/IFlightControlParameter.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Sometimes Git baffles me.  Exactly how is this a "change"?  The old and the new line are identical.